apache-maven/src/assembly/maven/conf/settings.xml

```diff
@@ -166,6 +166,38 @@ under the License.
     </mirror>
   </mirrors>
 
+  <!-- repositories
+   | Specifies the list of default remote repositories that maven will search artifacts for.
+  -->
+  <repositories>
+    <repository>
+      <id>central</id>
+      <name>Central Repository</name>
+      <url>https://repo.maven.apache.org/maven2</url>
+      <snapshots>
+        <enabled>false</enabled>
+      </snapshots>
+    </repository>
+  </repositories>
+
+  <!-- plugin repositories
+   | Specifies the list of default remote repositories that maven will search plugins for.
+  -->
+  <pluginRepositories>
+    <pluginRepository>
+      <id>central</id>
+      <name>Central Repository</name>
+      <url>https://repo.maven.apache.org/maven2</url>
+      <snapshots>
+        <enabled>false</enabled>
+      </snapshots>
+      <releases>
+        <updatePolicy>never</updatePolicy>
```

Comment:
why never? why the diff between central repo and central plugin repo?

Reply:
That's the current value, unchanged. 
See https://github.com/apache/maven/pull/1142/commits/04268f9dd323dd31630a55004ee4ca10e412c360 for the follow-up commit which changes the value.